sky/skylet/constants.py

```diff
@@ -66,19 +74,38 @@
     DOCKER_SERVER_ENV_VAR,
 }
 
+ACTIVATE_PYTHON_ENV = (f'[ -d {SKY_REMOTE_PYTHON_ENV} ] && '
```

Comment:
In what case will the remote python env not exist? Should we print some warning here?